### PR TITLE
Configure mouse position w/ an object w/ target mandatory

### LINKED ISSUE #1323: Affichage des coordonnées de la souris
Propriété `mouseposition` qui prend comme valeur l'identifiant de l'élément dans lequel les coordonnées s'afficheront.

Question : système de projection des coordonnées à définir.

### PR DESCRIPTION
**Pull request by pgiraud**
Fixes #1323.

### CODE REVIEW

File: jsapi/src/main.js [hunk 1]
```diff
@@ -94,10 +94,15 @@ lux.Map = function(options) {
...
   var controls = ol.control.defaults();
-  var srs = options.mousePositionSrs ?
-      'EPSG:' + options.mousePositionSrs.toString() : 'EPSG:2169';
   if (options.mousePosition) {
+    var srs = options.mousePosition.srs ?
+        'EPSG:' + options.mousePosition.srs.toString() : 'EPSG:2169';
+    var target = options.mousePosition.target;
+    var el = typeof target === 'string' ?
+         document.getElementById(target) :
+         target;
```
Comment (tonio) [inline jsapi/src/main.js]: Shouldn’t we ensure `el` is an element, and log error instead?

File: jsapi/externs/luxx.js [hunk 2]
```diff
@@ -136,3 +129,25 @@ luxx.LayerOptions;
...
  */
 luxx.LayersOptions;
+
+
+/**
+ * Position of the marker. If not set, the marker is displayed at the center of
+ * the map.
+ * @type {ol.Coordinate|undefined}
+ */
+luxx.MarkerOptions.prototype.position;
```
Comment (tonio) [inline jsapi/externs/luxx.js]: Not related. Was it missing?

File: jsapi/src/main.js (hunk 1 above)
Comment (pgiraud) [inline jsapi/src/main.js]: Done.

File: jsapi/externs/luxx.js [hunk 3]
```diff
@@ -136,3 +129,17 @@ luxx.LayerOptions;
  * @typedef {Object.<number|string, luxx.LayerOptions>}
  */
 luxx.LayersOptions;
+
+/**
+ * Object literal with config options for the maker.
```
Comment (tonio) [inline jsapi/externs/luxx.js]: `s/maker/mouse position control/`

File: jsapi/externs/luxx.js [hunk 4]
```diff
@@ -136,3 +129,17 @@ luxx.LayerOptions;
...
+ *     srs: (number|string)}}
+ */
+luxx.MousePositionOptions;
+
+ /**
+ * The projection of the mouse position control.
+ * Default is `2169`.
+ * @type {string|number}
+ */
+luxx.MousePositionOptions.prototype.srs;
```
Comment (tonio) [inline jsapi/externs/luxx.js]: missing `target` ?

Comment (pgiraud) [inline jsapi/externs/luxx.js]: Fixed.

**Comment by tonio** [comment]
Still wrong definitions in `MapOptions` :

```
 *     mousePosition: (boolean),
 *     mousePositionSrs: (string|number|undefined)
```

**Pull request merged**